src/pal/src/thread/context.cpp

```diff
@@ -551,6 +560,15 @@ void CONTEXTFromNativeContext(const native_context_t *native, LPCONTEXT lpContex
         }
 #endif
     }
+
+    // TODO: Enable for all Unix systems
+#if defined(_AMD64_) && defined(LINUX64)
+    if ((contextFlags & CONTEXT_XSTATE_YMMH) != 0 && FPREG_HasExtendedState(native))
+    {
+        lpContext->ContextFlags |= CONTEXT_XSTATE_YMMH;
```

Comment:
This doesn't seem to be needed, the lpContext->ContextFlags are assigned the contextFlags at the beginning of the function.